tests/queries/0_stateless/02722_database_filesystem.sh

```diff
@@ -35,13 +41,20 @@ echo $?
 ${CLICKHOUSE_CLIENT} --query "SHOW DATABASES" | grep "test1"
 ${CLICKHOUSE_CLIENT} --query "SELECT COUNT(*) FROM test1.\`${unique_name}/tmp.csv\`;"
 ${CLICKHOUSE_CLIENT} --query "SELECT COUNT(*) FROM test1.\`${unique_name}/tmp/tmp.csv\`;"
+${CLICKHOUSE_CLIENT} --query "SELECT COUNT(*) FROM test1.\`${unique_name}/tmp_numbers_*.csv\`;"
+${CLICKHOUSE_CLIENT} --query "SELECT COUNT(*) FROM test1.\`${unique_name}/tmp/*tmp_numbers_*.csv\`;"
+${CLICKHOUSE_CLIENT} --query "SELECT COUNT(*) FROM test1.\`${unique_name}/*/*tmp_numbers_*.csv\`;"
```

Comment:
lets also add this case
```
SELECT _path FROM test1.\`*\`;
```
Here `_path` is a virtual column, which is added automatically.
And check that in the result all paths start with user_files_dir.
This can probably be done in the query itself:
```
SELECT count() FROM test1.\`*\` WHERE startsWith(_path, "${user_files_tmp_dir}");
SELECT count() FROM test1.\`*\` WHERE not startsWith(_path, "${user_files_tmp_dir}");
```

Reply:
Done!

```
${CLICKHOUSE_CLIENT} --query "SELECT count(*) FROM test1.\`*/*.csv\` WHERE startsWith(_path, '${user_files_tmp_dir}')";
${CLICKHOUSE_CLIENT} --query "SELECT count(*) FROM test1.\`*/*.csv\` WHERE not startsWith(_path, '${user_files_tmp_dir}')";
```

Small changes:
- `*` glob doesn't recursively look into nested directories. I changed that to '*/*' to o make lookup recursive.

- Added `.csv` to the search pattern because file type is computed from the extension. Without `.csv` this error happens: `Code: 36. DB::Exception: Cannot determine the file format by its extension` 